satpy/tests/reader_tests/test_generic_image.py

```diff
@@ -209,8 +209,7 @@ def test_GenericImageFileHandler(self):
 
         dataset = reader.get_dataset(foo, {})
         self.assertTrue(isinstance(dataset, xr.DataArray))
-        self.assertIn('crs', dataset.attrs)
-        self.assertIn('transform', dataset.attrs)
+        self.assertIn('spatial_ref', dataset.coords)
```

Comment:
Does this change from `crs` and `transform` require any changes to other parts so that resampling still works?

Reply:
I'm not familiar with this reader and how the data interacts with the rest of satpy. However, I can see that the reader has a `get_area_def` method, so I don't expect this attribute to be used anywhere else. Hopefully someone using this reader can help me here.